pkg/sql/catalog/bootstrap/initial_values.go

```diff
@@ -90,9 +104,191 @@ func buildLatestInitialValues(
 ) (kvs []roachpb.KeyValue, splits []roachpb.RKey, _ error) {
 	schema := MakeMetadataSchema(opts.Codec, opts.DefaultZoneConfig, opts.DefaultSystemZoneConfig)
 	kvs, splits = schema.GetInitialValues()
+
+	if !roachpb.EnableExperimentalUA && opts.Codec.TenantID == roachpb.TenantTwo {
+		copiedKVs, err := copySystemTableKVs(context.Background(), opts.Txn, opts.IsqlTxn, opts.Codec, schema.descsMap)
+		if err != nil {
+			return nil, nil, err
+		}
+		kvs = append(kvs, copiedKVs...)
+	}
+
 	return kvs, splits, nil
 }
 
+func copySystemTableKVs(
+	ctx context.Context,
+	kvTxn *kv.Txn,
+	isqlTxn isql.Txn,
+	targetCodec keys.SQLCodec,
+	descMap map[string]catalog.Descriptor,
+) ([]roachpb.KeyValue, error) {
+	ret := make([]roachpb.KeyValue, 0)
+	sourceCodec := keys.SystemSQLCodec
+
+	if kvs, err := systemTenantTableKVs(
+		ctx, kvTxn, isqlTxn, targetCodec, descMap); err != nil {
+		return nil, err
+	} else {
+		ret = append(ret, kvs...)
+	}
+
+	tables := []catconstants.SystemTableName{
+		catconstants.ZonesTableName, // we need to filter out data for tables we don't need
+		catconstants.TenantSettingsTableName,
+		// Latest value for `tenant_id_seq` can be read using `.Scan` because in
+		// `updateTenantIDSequence` we use `.Put` to update instead of
+		// `InternalExecutor`.
+		catconstants.TenantIDSequenceTableName,
+		catconstants.RoleIDSequenceName,
+	}
+
+	batch := kvTxn.NewBatch()
+	for _, table := range tables {
+		desc, ok := descMap[string(table)]
+		if !ok {
+			log.Ops.Errorf(ctx, "descID not found for : %s", table)
+			return nil, errors.Errorf("descID not found for : %s", table)
+		}
+		span := sourceCodec.TableSpan(uint32(desc.GetID()))
```

Comment:
- [ ] TODO(shubham): Copy wouldn't work for tables with dynamic desc id tables for some clusters. Fix this.